maven-publish.gradle

```diff
@@ -0,0 +1,78 @@
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+
+task javadocsJar(type: Jar, dependsOn: "dokkaJavadoc") {
+    getArchiveClassifier().set("javadoc")
+    from dokkaJavadoc.outputDirectory
+}
+
+task sourcesJar(type: Jar) {
+    getArchiveClassifier().set("sources")
+    from sourceSets.main.allJava
+}
+
+
+afterEvaluate {
+
+    publishing {
+        def properties = project.ext.mavenPublishProperties
+
+        repositories {
+            maven {
+                name "SonatypeMavenCentral"
+                url properties.repository.url
+                credentials {
+                    username properties.repository.username
+                    password properties.repository.password
+                }
+            }
+        }
+        publications {
+            release(MavenPublication) {
+                from components.java
+
+                groupId = properties.group
+                version = properties.version
+                artifactId = properties.artifactId
+
+                artifact sourcesJar
+                artifact javadocsJar
+
+                pom {
+                    name = properties.name
+                    description = properties.description
+                    url = properties.url
+                    licenses {
+                        license {
+                            name = 'The Apache License, Version 2.0'
+                            url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                        }
+                    }
+                    developers {
+                        developers {
+                            developer {
+                                id.set("antunflas")
+                                name.set("Antun Flaš")
+                                email.set("antun.flas@infinum.com")
+                            }
+                        }
+                        developer {
+                            id = 'thisAAY'
+                            name = 'Ahmed Ali'
+                            email = 'ahmedali.abdelghany.yousef@infinum.com'
+                        }
+                    }
```

Comment:
It's not really a blocker or big thing. But we should replace personal info with company's info.

```
organization {
    name = 'Infinum Inc.'
    url = 'https://infinum.com'
}
developers {
        developer {
        id = 'Infinum'
        name = 'Infinum Inc.'
        url = 'https://infinum.com'
    }
}
```

Reply:
Should we do this for all libs? because i think this is the same setup for JsonApiX too

Reply:
Yes, but it is not big priority, you can fix it on JsonApiX when you will do some work there